conf/application.conf

```diff
@@ -104,7 +104,7 @@ microservice {
       host = "http://localhost:9232"
     }
 
-    bas-gateway {
+    bas-gateway-frontend {
```

Comment:
does app-config-qa, app-config-staging & app-config-production need to be amended or removed?

Reply:
In the comments for this PR there are the other PRs that need to be merged when this one is. (app-config-base and the journey tests). The app-config base has the change.